Lib/test/clinic.test.c

```diff
@@ -4948,3 +4948,119 @@ Test_meth_coexist(TestObj *self, PyObject *Py_UNUSED(ignored))
 static PyObject *
 Test_meth_coexist_impl(TestObj *self)
 /*[clinic end generated code: output=808a293d0cd27439 input=2a1d75b5e6fec6dd]*/
+
+
+/*[clinic input]
+test_deprecate_positional_use
+    pos: object
+    * [from 3.14]
+    optarg: int = 5
+[clinic start generated code]*/
```

Comment:
And let's test how this works with regular `*`:

```
pos: object
* [from 3.14]
optarg: int
*
kw: int
```

Reply:
Added, and this test case does not produce correct code. I'll work on it.

Reply:
Fixed